tests/test_mapmaker_pointing.py

```diff
@@ -26,7 +26,10 @@
 if toast_available:
     import healpy as hp
 
-from ._helpers import calibration_schedule, close_data_and_comm, simulation_test_data
+try:
+    from . import _helpers as helpers
+except ImportError:
+    import _helpers as helpers
```

Comment:
Why would the relative import fail?  We want to make sure we are always importing the `_helpers` module located in the `tests` directory, not in other locations.

Reply:
With this modification, I can execute this one unit test individually with ```python tests/test_mapmaker_pointing.py```.

Reply:
Instead of just doing
```
python -m unittest tests/test_mapmaker_pointing.py
```
?

Reply:
yes

Reply:
reverted